www/js/history/review-controller.js

```diff
@@ -2,13 +2,50 @@ angular.module('starter.review', [])
 
 .controller('ReviewCtrl', function($scope, $stateParams) {
 
+	$scope.newReview = {};
+
+// START: Five star rating thang
+	// Initialization code
+	var fiveStarElement = document.getElementById("fiveStarRating");
+	var fourStarElement = document.getElementById("fourStarRating");
+	var threeStarElement = document.getElementById("threeStarRating");
+	var twoStarElement = document.getElementById("twoStarRating");
+	var oneStarElement = document.getElementById("oneStarRating");
```

Comment:
yeah i see what you mean. i have a feeling there might be a better way to do it, but i can't think of a better solution off the top of my head. we can keep it as is